tests/test_middleware.py

```diff
@@ -9,24 +21,34 @@
 from .base import HostsTestCase
 
 
+def mock_get_response(request):
```

Comment:
In `django/tests/middleware/tests.py`, a similar method is called `get_response_empty` so I'd suggest that name.